scripts.js

```diff
@@ -0,0 +1,123 @@
+var newGameBtn = document.getElementById('js-newGameButton');
+
+newGameBtn.addEventListener('click', newGame);
+
+var pickRock = document.getElementById('js-playerPick_rock');
+    pickPaper = document.getElementById('js-playerPick_paper');
```

Comment:
deklaracja zmiennej powinna być poprzedzona `var` - inaczej wprowadzasz do skryptu globalne zmienne, to bardzo nieporządane